### PR TITLE
Documentation & Cog Panel Layouts for 8.x-1.x

### PR DESCRIPTION
**Pull request by jenter**

### CODE REVIEW

File: STARTERKIT/package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
   "private": true,
   "//": "The postinstall script is needed to work-around this Drupal core bug: https://www.drupal.org/node/2329453",
   "scripts": {
+    "install-tools": "yarn install;",
```
Comment (skippednote) [inline STARTERKIT/package.json]: Do we need to alias `yarn` to `npm run install-tools`?
It's 17 more keypresses for every developer.

Comment (jenter) [inline STARTERKIT/package.json]: @skippednote true :) we can leave it out, but we might want to address for the BLT phing tasks. Have the BLT build tasks been updated for yarn yet? 

cc @sarahjean @aellison

Comment (skippednote) [inline STARTERKIT/package.json]: No, it hasn't. And I don't it should be updated to directly use yarn. We nee to check check if yarn is available else fallback to npm. I'll take this up, the changes would go in Cog only.

Comment (jenter) [inline STARTERKIT/package.json]: @skippednote great, ok thanks man

**Comment by skippednote** [comment]
@jenter `blt` doesn't install Cog by default and there are no specific front-end tasks pre-configured. So right now all be need the user to do run `yarn` in the sub-theme to install the dependencies.

**Comment by sarahjean** [comment]
@skippednote blt does install cog, but it doesn't automatically create a subtheme at the moment. You are right that there will need to be some blt config done after you create your subtheme to get everything ready to go.

File: STARTERKIT/README.md [hunk 2]
```diff
@@ -1,8 +1,62 @@
...
+----------------
+
+
+## Installation
+
+### Create Cog Sub-Theme
+
+* In your `themes/` directory create the `contrib/` and `custom/` directories
+* Download Cog into the `themes/contrib` folder and enable using `drush en cog`
+* Set Cog as your default theme `drush config-set system.theme default cog`
```
Comment (sarahjean) [inline STARTERKIT/README.md]: I was testing earlier, it seems like themes just need to be enabled in order to use drush commands.

File: STARTERKIT/README.md [hunk 3]
```diff
@@ -1,8 +1,62 @@
...
+* Set `MyTheme` as your default theme `drush config-set system.theme default mytheme`
+
+Available options for creating your new theme `drush help cog`
+
+### Setup Local Development
+
+Once you have created a custom sub-theme, you will setup for local compiling. If you would like to review an extension explanation of these steps, read the [full setup readme](_readme/setup-full.md).
+
+* Navigate to `themes/custom/mytheme` folder in your terminal
+* Install Node.js with `./install-node.sh 4.4.4` and then point to the proper version with `source ~/.bashrc && nvm use --delete-prefix 4.4.4`
```
Comment (jenter) [inline STARTERKIT/README.md]: @sarahjean let's coordinate another PR once this once is done then.

**Comment by jenter** [comment]
@sarahjean have you had a chance to review yet?

File: README.md [hunk 4]
```diff
@@ -1,15 +1,23 @@
-## Getting started with Cog
+## Cog
 
-The Cog documentation you seek is now located in the following directory: `STARTERKIT/README.md`. This documentation will cover setup steps with Cog and overall D8 theme documenation with code examples.
+Cog is a developer-focused base theme and starterkit created by Acquia's Professional Service FrontEnd Team. It is intended as a minimalistic baseline for custom theming, while exposing common tools and workflows. Cog provides a small amount of code to get started, but is still packed with utilities to extend.
```
Comment (skippednote) [inline README.md]: The document uses **front-end**, **front end**, and **frontend**.
We need to make this consistent.

File: STARTERKIT/README.md (hunk 2 above)
Comment (grasmash) [inline STARTERKIT/README.md]: Yeah I was wondering about this too. Since you're just going to be changing the default theme in a moment, what's the point?

**Comment by jenter** [comment]
@sarahjean @aellison adjustments finished, let me know if anything else needs to change before merge.

**Pull request merged**